scripts/generate-method-action-types.ts

```diff
@@ -0,0 +1,575 @@
+#!yarn ts-node
+
+import { ESLint } from 'eslint';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+import yargs from 'yargs';
+
+const eslint = new ESLint({
```

Comment:
Nit: Instead of setting a global, do we want to create this in `main` and then pass this down? That function sets up the rest of the script.